### PR TITLE
TextArea: updates story from knobs to control

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to control.
**Which issue(s) this PR fixes**:

Relates to #30182

### CODE REVIEW

File: packages/grafana-ui/src/components/TextArea/TextArea.story.tsx [hunk 1]
```diff
@@ -12,36 +12,27 @@ export default {
...
-    CONTAINER_GROUP
-  );
+  },
+  argTypes: {
+    cols: { control: { type: 'range', min: 5, max: 50, step: 5 } },
+    containerWidth: { control: { type: 'range', min: 100, max: 500, step: 10 } },
+  },
+} as Meta;
 
+export const Simple: Story = (args) => {
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/TextArea/TextArea.story.tsx:25]: Maybe we should follow the convention and call the first story `Basic`?
```suggestion
export const Basic: Story = (args) => {
```

**Review by hugohaggmark — APPROVED** [review]
Looks good and works as expected 🎉  Not sure what value the `cols` prop gives to the story so for me you go ahead and remove it.

File: packages/grafana-ui/src/components/TextArea/TextArea.story.tsx (hunk 1 above)
Comment (thisisobate) [reply packages/grafana-ui/src/components/TextArea/TextArea.story.tsx:25]: Alright. 👍

**Comment by hugohaggmark** [comment]
@thisisobate merge with main to fix the failing backend tests, a fix for them was merged recently.

**Pull request merged**